lib/validator.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = {
+  auth: Joi.object().keys({
+    credentials: Joi.object().required()
+  }),
```

Comment:
I think this was trying to prevent the plugin from running the db query if authentication failed, though it seems that  "[the presence of an object does not mean successful authentication](http://hapijs.com/api/11.1.2#request-object)." So if we correctly checked for an authenticated request using `auth.isAuthenticated`, should that still be the responsibility of the `filter` function on the model?